google/cloud/security/inventory/inventory_loader.py

```diff
@@ -47,7 +47,7 @@
 from google.cloud.security.common.data_access import errors as data_access_errors
 from google.cloud.security.common.data_access.dao import Dao
 from google.cloud.security.common.data_access.sql_queries import snapshot_cycles_sql
-from google.cloud.security.common.gcp_api import admin_directory
+from google.cloud.security.common.gcp_api import admin_directory as ad
```

Comment:
update -- here's the link from the style guide https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports

"Use imports for packages and modules only"